addons/website_slides_survey/models/survey_survey.py

```diff
@@ -22,6 +23,12 @@ def _compute_slide_channel_data(self):
             survey.slide_channel_ids = survey.slide_ids.mapped('channel_id')
             survey.slide_channel_count = len(survey.slide_channel_ids)
 
+    @api.ondelete(at_uninstall=False)
+    def _unlink_except_linked_to_course(self):
```

Comment:
Would be nice to add a test about this :)

Reply:
I am adding the test right now.